.travis.yml

```diff
@@ -33,8 +33,8 @@ before_install:
 
 install:
   - composer install
-  - npm install google-closure-library
+  - cd Resources && npm install
```

Comment:
running phpunit is now broken, because of the ``cd Resources`` you added here, as you don't come back to the root after that